### ISSUE TITLE
Can't install Oracle jdk from AUR

### ISSUE THREAD

**Issue by carymrobbins** [aurapm/aura#429]
```
% uname -vr
4.8.13-1-ARCH #1 SMP PREEMPT Fri Dec 9 07:24:34 CET 2016

% sudo aura -Ay jdk                                            
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
aura >>= Determining dependencies...
aura >>= AUR Packages:
jdk

aura >>= Continue? [Y/n] 
aura: /var/cache/aura/pkgbuilds/jdk.pb: openFile: does not exist (No such file or directory)
```

**Comment by carymrobbins** [comment]
Note that installing it by cloning the AUR repo and running `makepkg` manually worked fine.

**Comment by carymrobbins** [comment]
The same thing just happened with Intellij -

```
% sudo aura -Ay intellij-idea-ultimate-edition                      
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community                    3.7 MiB  72.8K/s 00:52 [############################] 100%
aura >>= Determining dependencies...
aura >>= AUR Packages:
intellij-idea-ultimate-edition

aura >>= Continue? [Y/n] 
aura: /var/cache/aura/pkgbuilds/intellij-idea-ultimate-edition.pb: openFile: does not exist (No such file or directory)
```

**Comment by carymrobbins** [comment]
I think doing this may solve the issue; however, I'd imagine that aura should do this for you -

```
% sudo mkdir /var/cache/aura/pkgbuilds
```

**Comment by fosskers** [comment]
You're right that the folder creation should occur automatically. This has come up before.

**Comment by fosskers** [comment]
This has been fixed and will be included in `aura-1.3.8`.
